Microsoft.HealthVault.IntegrationTest/ActionPlanTests.cs

```diff
@@ -55,16 +55,19 @@ public class ActionPlanTests
             var plans = await api.ActionPlans.GetAsync();
             foreach (var plan in plans.Plans)
             {
-                await api.ActionPlans.DeleteAsync(plan.Id);
+                if (plan.Id != null)
```

Comment:
So the Id here wasn't added to the Required list in the swagger file when it was assigned the UUID format, which resulted in a nullable Guid, even though I'm pretty sure it will never be null for ActionPlanInstances.

Thoughts?  We can file a bug for it.

Reply:
Created a bug for myself on this